artifacts/manifests/k8s-staging-sig-storage/master.yaml

```diff
@@ -0,0 +1,3 @@
+files:
+- name: 1.0.0/csi-proxy.exe
```

Comment:
We have released a 1.0.1 now right?

Reply:
right, thanks for the review. updated the version to 1.0.2, the latest.